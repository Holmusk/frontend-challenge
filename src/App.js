import React, {Component} from 'react';
import { Provider } from "react-redux";
import Home from './containers/home';

import store from './redux/store';

import './style.css';

export default class App extends Component {
    render(){
        return(
            <Provider store={store}>
                <Home />
            </Provider>
        )
    }
}

