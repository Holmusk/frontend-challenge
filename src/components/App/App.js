import Container from '../Container/Container.vue';

export default {
  name: 'App',
  components: {
    Container,
  },
};
