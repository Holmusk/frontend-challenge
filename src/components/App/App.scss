@import '../../styles/variables.scss';

html {
  height: 100%;
}

body {
  margin: 0;
  padding: 0;
  min-height: 100%;
  display: flex;
  flex-direction: column;
}

body, input, textarea, select, button {
  font-family: Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-size: 14px;
  font-weight: normal;
  color: $color-text;
}

#app {
  padding-top: 60px;
  background-color: $color-background;
  flex: 1;
  display: flex;
  justify-content: center;
  //align-items: center;
}
