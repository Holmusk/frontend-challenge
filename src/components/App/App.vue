<template>
  <div id="app">
    <Container />
  </div>
</template>

<script src="./App.js"></script>

<style src="./App.css"></style>
