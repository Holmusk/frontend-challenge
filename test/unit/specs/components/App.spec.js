import Vue from 'vue';

import App from '@/components/App/App.vue';

const propsData = {};

describe('App.vue', () => {
  it('should render correctly', () => {
    const Constructor = Vue.extend(App);
    const vm = new Constructor({ propsData }).$mount();
    expect(vm.$el).toMatchSnapshot();
  });
});
