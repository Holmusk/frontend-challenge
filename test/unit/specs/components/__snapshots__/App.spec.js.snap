// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue should render correctly 1`] = `
<div
  id="app"
>
  <div
    id="container"
  >
    <div
      class="app-controls"
    >
      <div
        class="manage-state"
      >
        <textarea
          id="serialized-state"
          name="serialized-state"
          placeholder="Please paste state here in JSON format"
        />
         
        <input
          type="button"
          value="Set state"
        />
         
        <input
          type="button"
          value="Copy state to clipboard"
        />
         
        <input
          type="button"
          value="Load state"
        />
      </div>
       
      <div
        class="create-group-form"
      >
        <input
          id="new-group-title"
          name="new-group-title"
          placeholder="Group title"
          type="text"
        />
         
        <input
          type="button"
          value="Create group"
        />
      </div>
       
      <!---->
    </div>
     
    <div />
  </div>
</div>
`;
